src/Sentry/Scope.cs

```diff
@@ -305,6 +309,28 @@ public void UnsetTag(string key)
     /// </summary>
     public void AddAttachment(Attachment attachment) => _attachments.Add(attachment);
 
+    /// <summary>
+    /// Resets all the properties and collections within the scope to their default values. 
+    /// </summary>
+    internal void Clear()
```

Comment:
```suggestion
    public void Clear()
```

Reply:
Added a new commit to address this and the missing docs on the ClearBreadcrumbs method